commit fc702888169abecd41b003a56122f857588e8f53
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2019-08-14T14:32:48-04:00 (8 weeks ago)
    
    MANTA-4502 node-fast provider transitive dependencies too old to build on x86_64

diff --git a/package.json b/package.json
index 67f9901..1979b31 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
 	"name": "fast",
 	"description": "streaming JSON RPC over TCP",
-	"version": "2.7.0",
+	"version": "2.8.0",
 	"main": "./lib/fast.js",
 	"repository": {
 		"type": "git",
@@ -15,7 +15,7 @@
 		"oldcrc": "git://github.com/joyent/node-crc.git#0.3.0-oldcrc",
 		"extsprintf": "^1.3.0",
 		"jsprim": "^1.2.2",
-		"kang": "^1.1.0",
+		"kang": "^1.3.0",
 		"lstream": "0.0.4",
 		"microtime": "2.1.6",
 		"posix-getopt": "1.2.0",
@@ -29,7 +29,7 @@
 		"restify": "5.2.0"
 	},
 	"optionalDependencies": {
-	    "dtrace-provider": "^0.8.5"
+	    "dtrace-provider": "^0.8.7"
 	},
 	"license": "MIT"
 }
